src/util.rs

```diff
@@ -0,0 +1,92 @@
+use crate::GraphFrame;
+use datafusion::arrow::datatypes::{DataType, Field, Schema};
+use datafusion::error::Result;
+use datafusion::prelude::{CsvReadOptions, SessionContext};
+use std::collections::HashMap;
+use std::io::Result as ioResult;
+use std::{env, fs};
+
+// Gets the basepath of the dataset based on if it's benchmark runs or test runs
+/// # Arguments
+///
+/// * `benchmark_run`: true for benchmark runs to read data from bench/data dir, false for tests to read data from testing/data.
+pub fn _get_dataset_base_path(benchmark_run: bool) -> Result<String> {
```

Comment:
Why is this function is pub and starts from the underscore at the same time? It is confusing